### DISCUSSION TITLE
[Bug report] External Links plugin mangles protocol in http links

### DISCUSSION THREAD

**Discussion by OttoKerner** [GetPublii/Publii#1641] [Bug Report] [answered]
### Operating system

Windows 11 23H2

### Publii version

0.46.1 (build 16918)

### Issue type

Free plugin

### Bug description

If there is an external link which uses the http protocol (instead of http**s**), the protocol is mangled in the rendered web page.
It appears as `httpundefined://[url]`

There is no issue when the link uses uses the https protocol.

### Steps to reproduce

Just set an external (User-Defined) link and use the http protocol, e.g. `http://getpublii.com`
Enable the "External Links" plugin.

**Comment by bobmitro** [comment, marked as answer]
Hey there! Thanks so much for bringing this to our attention!

I found the issue with HTTP links showing up as `httpundefined://[url]`, and I’ve fixed it!
The updated plugin version **2.0.1** is now available for [download](https://marketplace.getpublii.com/plugins/external-links/). Just grab the latest update, and everything should work perfectly. 

Thanks again for your help, and let us know if you run into anything else!
